src/containers/StudyEdit/StudyEditForm.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
+import {
+  MultiLineInput,
+  LabelInput,
+  FileInput,
+} from '@src/components/StudyCreate&Edit';
+import { fetchStudyDetails } from '@src/apis/fetchStudyDetails';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+import {
+  StudyEditHeading,
+  StudyEditWrapper,
+  StudyEditImageWrapper,
+  ImageWrapper,
+  Image,
+  ButtonWrapper,
+  StudyDescriptionWrapper,
+} from './style';
+
+function StudyEditForm() {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imageSrc, setImageSrc] = useState<File | null>(null);
+  const [thumbnailImage, setThumbnailImage] = useState<string>('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchStudyDetails();
+      const { title, description, imageUrl } = data;
+
+      setTitle(title);
+      setDescription(description);
+      setThumbnailImage(imageUrl);
+    };
+
+    fetchData();
+  }, []);
+
+  const encodeFile = (fileBlob: File) => {
+    const reader = new FileReader();
+    if (!fileBlob) return;
+    reader.readAsDataURL(fileBlob);
+
+    return new Promise<void>((resolve) => {
+      reader.onload = () => {
+        const result = reader.result as string;
+        setThumbnailImage(result);
+
+        resolve();
+      };
+    });
+  };
+
+  const onImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { files } = e.target;
+
+    if (!files || !files[0]) return;
+
+    const maxSize = 1 * 1024 * 1024;
+    const fileSize = files[0].size;
+
+    if (fileSize > maxSize) {
+      alert('첨부 파일은 최대 1MB 입니다.');
+      return;
+    }
+
+    setImageSrc(files[0]);
+    encodeFile(files[0]);
+  };
+
+  const handleDescription = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value = '' } = e.target;
+    setDescription(value);
+  };
+
+  const handleTitle = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value = '' } = e.target;
+    setTitle(value);
+  };
+
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log('제목 ' + title);
+    console.log('이미지 ' + imageSrc);
+    console.log('내용' + description);
+  };
+  return (
+    <form onSubmit={onSubmit}>
+      <StudyEditWrapper>
+        <StudyEditHeading>
+          <Typography variant='h4'>스터디 수정</Typography>
+          <LabelInput
+            id='study-title'
+            label='스터디명'
+            value={title}
+            fullWidth={true}
+            onChange={handleTitle}
+          />
+        </StudyEditHeading>
+        <StudyEditImageWrapper>
+          <Typography variant='h5'>대표 이미지</Typography>
+          <ImageWrapper>
+            {thumbnailImage && <Image src={thumbnailImage} alt='study-image' />}
+          </ImageWrapper>
```

Comment:
`material icons`에 있는 `CameraAltOutlinedIcon` 컴포넌트를 사용하면 figma와 동일한 `UI`를 만들 수 있을 것 같습니다!

일단은 빠르게 하려고 `sx`로 해두었는데, `style.ts` 파일에서 분리해서 수정하시면 더욱 깔끔할 것 같습니다.
```suggestion
    <ImageWrapper>
        <CameraAltOutlinedIcon
          color='secondary'
          sx={{
            position: 'absolute',
            top: '50%',
            left: '50%',
            transform: 'translate(-50%, -50%)',
          }}
        />
    {thumbnailImage && <Image src={thumbnailImage} alt='study-image' />}
    </ImageWrapper>
```

Reply:
이렇게 하면 이미지가 추가됐을 때도 `camera icon`이 썸네일과 동시에 보이게 되는데, 이 부분은 `thumbnailImage` state로 조건부 렌더링 하시면 될 것 같습니다!

Reply:
styles.ts에서 CameraIcon으로 분리해서 반영했습니다! 27790bf6f41c5f10a07a56fe0013529d292e9caf